storage/innobase/xtrabackup/test/t/pxb-1961.sh

```diff
@@ -0,0 +1,65 @@
+############################################################################
+# PXB-1961: xtraback fails with encryption='n' with Percona server
+############################################################################
+
+function backup_and_restore() {
+record_db_state test
+rm -rf $topdir/backup
+mkdir -p $topdir/backup
+# Backup
+xtrabackup --datadir=$mysql_datadir --backup --target-dir=$topdir/backup 
+# Prepare
+xtrabackup --datadir=$mysql_datadir --prepare --target-dir=$topdir/backup
+stop_server
+# Restore
+rm -rf $mysql_datadir/ibdata1 $mysql_datadir/ib_logfile* \
```

Comment:
any reason for selective delete here rather than remove the entire `$mysql_datadir` ?